.gitmodules

```diff
@@ -82,3 +82,6 @@
 [submodule "third_party/foxi"]
 	path = third_party/foxi
 	url = https://github.com/houseroad/foxi.git
+[submodule "third_party/tvm"]
```

Comment:
can we make the user do extra legwork here and actually clone TVM into the pytorch root directory.
I dont want this to be the default submodule that everyone has to clone (TVM and it's submodules end up being a significant chunk of git clone)

Reply:
Also, why are we using Facebook's fork?

Reply:
@apaszke it's got a patch that enables the C++ frontend (hasn't been landed into dmlc yet)